gcc/d/dfrontend/template.c

```diff
@@ -7442,22 +7444,25 @@ Dsymbols *TemplateInstance::appendToModuleMember()
     }
     //printf("\t--> mi = %s\n", mi->toPrettyChars());
 
-    Dsymbols *a = mi->members;
-    for (size_t i = 0; 1; i++)
+    if (memberOf == mi)     // already a member
     {
-        if (i == a->dim)
-        {
-            a->push(this);
-            if (mi->semanticRun >= PASSsemantic3done && mi->isRoot())
-                Module::addDeferredSemantic3(this);
-            break;
-        }
-        if (this == (*a)[i])    // if already in Array
+#ifdef DEBUG
```

Comment:
Is there some configure flag / environment variable to enable this? Or is this meant to be used with `CFLAGS=-DDEBUG`?